common/lib/dependabot/file_updaters/vendor_updater.rb

```diff
@@ -18,7 +18,9 @@ def updated_vendor_cache_files(base_directory:)
         return [] unless repo_contents_path && vendor_dir
 
         Dir.chdir(repo_contents_path) do
+          # rubocop:disable Performance/DeletePrefix
           relative_dir = Pathname.new(base_directory).sub(%r{\A/}, "").join(vendor_dir)
+          # rubocop:enable Performance/DeletePrefix
```

Comment:
This one may be skippable upstream too, and also `Pathname#delete_prefix` sounds like a nice feature request. I can take of requesting both if they make sense.

Reply:
Yeah, there's its complement `Performance/DeleteSuffix` has a similar problem.

It would be great if RuboCop didn't generate a false positive for `Pathname`. In https://github.com/rubocop/rubocop-performance/issues/245, they marked it as not safe, but I don't know of any other changes beyond that.

Conversely, it'd also be nice if `Pathname` responded to `#delete_prefix` and `#delete_suffix`. I was surprised those methods didn't exist already.